logstream/grafana.go

```diff
@@ -0,0 +1,47 @@
+package logstream
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const (
+	ShorteningFailedErr = "failed to shorten Grafana URL"
+)
+
+func ShortenUrl(grafanaUrl, urlToShorten, bearerToken string) (string, error) {
```

Comment:
Can we add some unit tests for this?

Reply:
I think that would be hard/overkill as we hit actual API here